esmvalcore/cmor/_fixes/cmip5/gfdl_cm2p1.py

```diff
@@ -56,6 +58,41 @@ def fix_data(self, cube):
         return cube
 
 
+class Sit(Fix):
+    """Fixes for sit"""
+
+    def fix_metadata(self, cubes):
+        """
+        Fix metadata.
+
+        Fixes bad bounds
+
+        Parameters
+        ----------
+        cube: iris.cube.Cube
+
+        Returns
+        -------
+        iris.cube.Cube
+
+        """
+        cube = self.get_cube_from_list(cubes)
+        time = cube.coord('time')
+        if time.bounds.max() > 1e8 and self.vardef.frequency == 'mon':
```

Comment:
Looking at one example file I found that this test for necessity is not reliable. For early times (~1850 in the historical) even the wrong bounds are below 1e8. Maybe a better test is to check whether the point lies inside its corresponding bounds?

Reply:
Can you point me to that file? I put this because in mine the values where of the order of 1e9 and I assume that all problems of this kind with the bounds will be in the same range.

Reply:
Sure: `sit_OImon_GFDL-CM2p1_historical_r1i1p1_186101-186512.nc`

Reply:
Thanks!

Reply:
It looks that they are using different units for the bounds (hours since 1850, maybe?), or more likely  that they converted the time units and forgot converting the time bounds